karavan-designer/src/designer/karavan.css

```diff
@@ -813,4 +818,8 @@
 }
 .karavan .knowledbase-eip-section .pf-v5-c-toggle-group{
     margin:16px;
+}
+.karavan .kamelet-section .kamelet-card .block-checkbox input{
+    width:18px;
```

Comment:
Please keep default PatternFly look-and-feel for checkbox.

Reply:
Noted, it looked too small that's why we tried increasing the size. can be removed.